src/internal/operators/startWith.ts

```diff
@@ -49,7 +49,7 @@ export function startWith<T>(...array: Array<T | SchedulerLike>): MonoTypeOperat
  * @method startWith
  * @owner Observable
  */
-export function startWith<T>(...array: Array<T | SchedulerLike>): MonoTypeOperatorFunction<T> {
+export function startWith<T, D>(...array: Array<T | SchedulerLike>): OperatorFunction<T, T | D> {
```

Comment:
I think the rest parameter signature should be made the same as the implementation signature. And I think both should be:

```ts
export function startWith<T, D = T>(...array: Array<D | SchedulerLike>): OperatorFunction<T, T | D>
```

Reply:
Yea I'll add it, but I didn't had the time to adjust and test it. Currently a bit busy, but I'll add it soon.